src/index.js

```diff
@@ -1,34 +1,76 @@
 import './style.css';
 
-const todoList = [
-  {
-    description: 'wake up early',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'setup for morning session',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Coding with microverse',
-    completed: true,
-    index: 2,
-  },
-];
-
-const displayTodos = (todo) => {
-  const ul = document.querySelector('ul');
-  let content = '';
-  todo.forEach((tod) => {
-    content += `<li class="item">
-        <input type="checkbox" />
-        <p contenteditable="true">${tod.description}</p>
-      </li>`;
+class Todo {
+  constructor(desc, completed, id) {
+    this.desc = desc;
+    this.completed = completed;
+    this.id = id;
+  }
+}
+class UI {
+  static displayTodos = (todos) => {
+    const ul = document.querySelector('ul');
+    if (!todos.length) ul.innerHTML = '';
+    let content = '';
+    todos.forEach((tod) => {
+      content += `
+        <li id=${tod.id} class="item">
+          <div class='group'>
+            <input class='bdan' name='checkbox' type="checkbox" />
+            <p contenteditable="true">${tod.desc}</p>
+          </div>
+          <i class="fa-solid fa-trash"></i>
+        </li>`;
+    });
     ul.innerHTML = content;
-  });
+    content = '';
+  };
 
-  content = '';
-};
-displayTodos(todoList);
\ No newline at end of file
+  static generateId = () => Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+
```

Comment:
Good job by implementing this method but in this project requirements you were asked to index list items using array length the first added item should have 1 as index the second one have 2 and so on

Reply:
@souad988  implementing this method but in this project requirements has been resolved by putting theminto array and fix the requirement .